### ISSUE TITLE
Bundled utility class names may cause conflicts if already in use

### ISSUE THREAD

**Issue by tomkiss** [nystudio107/craft-code-editor#8]
The plugin includes a CSS file on the front end that uses some utility class names that can conflict with existing class names in a project.

Here's a stripped down example of what can happen in a front-end template that loads the code editor.

```html
<!-- css declared, via a project's main stylesheet -->
<style>
.hidden {
  display: none;
}
@media(min-width: 900px) {
  .md\:block {
    display: block;
  } 
}
</style>

<div class="hidden md:block">Desktop only!</div>

<!-- css declared, via the plugin's styles.css -->
<style>
.hidden {
  display: none;
}
</style>
```

In this example, the plugin's CSS redefines the `.hidden` class, making the `div` hidden at the medium breakpoint.
This happens with other classes in the plugin too.

**Comment by khalwat** [comment]
Same as: https://github.com/nystudio107/craft-code-editor/issues/9

**Comment by tomkiss** [comment]
It appears to come from [this line](https://github.com/nystudio107/craft-code-editor/blob/46bcd7a0e468da5af4cca0d047320ce7bab0fc93/src/web/assets/src/css/code-editor.pcss#L6), which generates the following CSS after its been built:

```css
/*!
 * @project        code-editor
 * @name           styles.css
 * @author         Andrew Welch
 * @build          Fri Dec 01 2023 03:11:47 GMT+0000 (Coordinated Universal Time)
 * @copyright      Copyright (c) 2023 ©2022 nystudio107.com
 *
 */
.fixed {
    position: fixed
}

.relative {
    position: relative
}

.box-content {
    box-sizing: content-box
}

.contents {
    display: contents
}

.hidden {
    display: none
}

.h-full {
    height: 100%
}

.transform {
    transform: translate(var(--tw-translate-x),var(--tw-translate-y)) rotate(var(--tw-rotate)) skewX(var(--tw-skew-x)) skewY(var(--tw-skew-y)) scaleX(var(--tw-scale-x)) scaleY(var(--tw-scale-y))
}

.p-2 {
    padding: .5rem
}
...
```

If you were to redeclare these classes manually with, an additional parent selector specific to the plugin, it should resolve the issue.

**Comment by tomkiss** [comment]
@khalwat Any thoughts on this? Although I do not know how the project build works, or have an understanding of the selectors required, I could probably do a PR if it would be helpful.

**Comment by khalwat** [comment]
Oh wow, I didn't realize it was coming from my CSS! My bad, sorry.

**Comment by tomkiss** [comment]
Amazing, thank you!

**Comment by khalwat** [comment]
Fixed in version `1.0.18` -> https://github.com/nystudio107/craft-code-editor/releases/tag/1.0.18

You can update via:

```
composer update nystudio107/craft-code-editor
```
